mod_reforged/hooks/entity/tactical/enemies/bandit_leader.nut

```diff
@@ -101,6 +113,73 @@
 		}
 	}
 
+	q.makeMiniboss = @() function()
+	{
+		if (!this.actor.makeMiniboss())
+		{
+			return false;
+		}
+
+		this.getSprite("miniboss").setBrush("bust_miniboss");
+		local shields = clone ::Const.Items.NamedShields;
```

Comment:
Because shields are only rolled in the `if <= 50` block therefore it would make sense to move this local variable declaration and the extension into that if block.